frontend/CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 All notable changes to this project will be documented in this file.
 
+## [Unreleased]
+
+## [1.7.12] - 2025-04-10
```

Comment:
We are already at version `1.7.14` based on `package.json` file.

So either you update the changelog and keep your changes in the `unreleased` section.

```suggestion

```

When you want to release the front-end/back-end, you bump the version in the `package.json` file (or `Cargo.toml` file for the backend). Tag the repo on the main branch for that release version and release the docker containers to Docker Hub. See the `READ.me` description at the top-level repo.